recipes/traja/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "traja" %}
+{% set version = "0.0.2a2" %}
```

Comment:
Pre release (alpha/beta) versions are normally not added to `conda-forge`.

I see there is a `0.0.4` release on PyPI and a `0.0.7` release on GitHub, could you update it?

Reply:
Updated PyPI to `0.0.7`, thank you.

Reply:
You also need to update the hash of the tarball, see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L19-L22).

Reply:
Updated, thanks!